packages/core/src/customer/strategies/stripe-upe/stripe-upe-customer-strategy.ts

```diff
@@ -98,8 +104,24 @@ export default class StripeUPECustomerStrategy implements CustomerStrategy {
                         },
                     },
                 };
-            } else {
-                appearance = {};
+            }
+
+            if (features && features['CHECKOUT-6879.enable_floating_labels']) {
+                appearance = {
+                    ...appearance,
+                    labels: 'floating',
+                    variables: {
+                        ...appearance?.variables,
+                        fontSizeBase: '14px',
+                    },
+                    rules: {
+                        ...appearance?.rules,
+                        '.Input': {
+                            ...appearance?.rules?.['.Input'],
+                            padding: '7px 13px 5px 13px',
```

Comment:
Where are we getting this numbers? Also can see the same repeated in shipping strategy...Can we make this a constant and use it that way?

Reply:
@animesh1987 I got it from the elements of BC (left and right), and for the top and bottom I had to adjust manually in order to emulate the BC styles since the iframe does not look similar using the same pixels of BC,

by the way, I moved it into a function to use in customer, shipping, and payment steps, let me know your thoughts on this, tomorrow I'll try to check the approach (including the JS approach), and QA will check it again